roles/build-release/README.md

```diff
@@ -4,12 +4,8 @@ An Ansible role that builds an Ansible release package with Antsibull and tests
 
 ### Requirements
 
-This role is expected to run directly from an antsibull git repository checkout.
```

Comment:
Where would the the role be if not in the git repo ? Should we consider moving it to it's own repo/collection ?

Reply:
The role is at the same place where the README.md file is that the person is reading. This could be a git repository checkout, an extracted version of a tarball downloaded from GItHub, or anything else.

Reply:
Oh, fair enough.